### ISSUE TITLE
Query

### ISSUE THREAD

**Issue by Light7654** [LagradOst/CloudStream-3#442]
Is the app only made in kotlin aur any other language also involved

Cause I am learning kotlin and wanna contribute my skill to this app in future

**Comment by LagradOst** [comment]
Yes, kotlin is the only programing language, (((however for UI android uses xml)))

**Comment by Kaluku10** [comment]
Where can u learn kotlin

**Comment by ITADORI-15** [comment]
YouTube,udemy Ig they r good sources 2 learn

**Comment by Light7654** [comment]
> Where can u learn kotlin

From many different sources - YouTube, skillshare, etc

**Comment by ITADORI-15** [comment]
YouTube is free....but udemy and skillshare cost some money

**Comment by Light7654** [comment]
> YouTube is free....but udemy and skillshare cost some money

There are many websites where you can download skillshare or any other paid course for free


In short piracy
